### DISCUSSION TITLE
[A-L] (2023/24) Foglio 2 - Esercizio 7

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#170] [Esercizi - Viaggi/Piazza]
![Es 7](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/89a971e5-e80f-4994-9057-133ca01424a3)

**Comment by Elia-Belli** [comment]
- **(7.1)** $a \cdot (-b) = - (a \cdot b)  = (-a) \cdot b$
   - Dimostriamo $a \cdot (-b) = - (a \cdot b)$:
Per la proprietà distributiva di $\cdot$ : $a(-b) + a \cdot b = a(-b+b) = a \cdot 0=0$, allora $a(-b)$ è inverso additivo per $(a\cdot b)$ ma lo è anche $-(a\cdot b)$, allora $a(-b)=-(a\cdot b)$ per unicità dell'inverso.
   - Dimostriamo analogamente $- (a \cdot b) = (-a) \cdot b$:
per la proprietà distributiva di $\cdot$ : $(-a) \cdot b + (a \cdot b)= (-a+a)\cdot b =(0)\cdot b=0$, allora $(-a)\cdot b)$ è inverso additivo per $(a\cdot b)$ ma lo è anche $-(a\cdot b)$, allora $-(a\cdot b)=(-a)\cdot b$ per unicità dell'inverso.
   - Allora $a \cdot (-b) = (-a) \cdot b$.

- **(7.2)** $(-a)\cdot (-b) -(a\cdot b) = (-a)\cdot (-b) +(-a)\cdot b = (-a)\cdot (-b+b) = (-a)\cdot 0 = 0$, allora $(-a)\cdot (-b)$ è inverso per $-(a\cdot b)$ ma lo è anche $(a\cdot b)$ allora $(-a)\cdot (-b)= a \cdot b$ per unicità dell'inverso.

- **(7.3)** $a \cdot (b-c) = a \cdot b - a \cdot c$
 $a \cdot (b-c) = a \cdot b + (a \cdot (-c))$, che per la **(7.1)** diventa $a\cdot b + (-(a\cdot c))= a \cdot b - a \cdot c$

**Comment by CiottoloMaggico** [comment]
analogo @Elia-Belli
